add the reusable footer part

lario0913 opened the pull request:
![screen](https://user-images.githubusercontent.com/26335055/67776966-61bc1a80-fa61-11e9-9fe0-3b0b40958c53.png)
![mobile](https://user-images.githubusercontent.com/26335055/67776967-61bc1a80-fa61-11e9-9ec9-8fb0dc4c8a71.png)

Joggz committed:
adds project structure:
- uses create-react-app
- uses bootstrap as a css framework
- uses sass preprocessor for styling
- installed radium incase of any inline styling

Joggz committed:
modifies project structure:
- adds components folder in src folder, all Components created should sit in this folder
- adds assest folder for assest like images to be used in the project
- adds container folder.
- adds README
- changes the import directory

Joggz committed:
Update README.md

Joggz committed:
Update README.md

made a change on line 1

Joggz committed:
Update README.md

Joggz committed:
adds component and assest folder:
- component and assest folder did not appear on github on my first push

Joggz committed:
deletes the SCSS folder from the project structure:
- each component should contain its own scss file

Joggz committed:
mades some changes to the readme

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

lario0913 committed:
completed the reusable footer part

SEUNAGBEYE commented on dae-alright/src/components/Footer/Footer.scss:
```diff
@@ -0,0 +1,149 @@
+
+
+        
+        .row1col1, .row2col1{
```
Good job so far, I love what I'm seeing. But I think you can do better with your `class names`. When I see a CSS `classname`, I want to know what it's for just by looking at the name. Unfortunately, `row2col1`, `row1col2` all don't have that effect.

lario0913 replied:
class names have been adjusted to reflect what they do..... here is the commit link
https://github.com/devcareer/Dae-Alright-Frontend/pull/4/commits/67e309ab69263f98ed160d24871f1bfd48f3ad68

John-pels committed:
SignUp Modal all done

lario0913 committed:
class names adjusted to reflect what they do

lario0913 committed:
class names adjusted to reflect what they do

lario0913 committed:
class names adjusted to reflect what they do

lario0913 committed:
class names adjusted to reflect what they do

lario0913 committed:
class names adjusted to reflect what they do

lario0913 committed:
class names adjusted to reflect what they do

John-pels committed:
I create the modal component folder

John-pels committed:
I create the reusable custom modal and also refactor the sign-up modal

Joggz committed:
restructured project folder

John-pels committed:
Working on the hover on the button

John-pels committed:
I resolve the conflicts

Joggz committed:
change directory to App.css in index.js and App.js

Joggz committed:
changes file directory

Joggz committed:
returns App.css App.js, App.test.js,index.css, index.js back into the src folder

Joggz committed:
changes directory for serviceWorker.js in index.js

Joggz committed:
deletes readme and gitignore

SEUNAGBEYE reviewed (commented):
This looks good to me. Get others to review and you need to rebase to fix your merge conflicts.

John-pels committed:
I remove the button

John-pels committed:
I add propsType to the component

Joggz committed:
adds reusuable inputs and buttons

lario0913 committed:
updated branch with the new folder structure

lario0913 committed:
updated branch with the new folder structure

John-pels committed:
I rename the modal components

lario0913 committed:
remote changes merged

Mlg29 reviewed (commented):
-Nice job, but the select element text suppose to be "See all Cities" and not select all cities
- Secondly, increase the height of the appstore and google store
-Thirdly, Terms and conditions and privacy policy isn't included in your footer
-Lastly, try to make it responsive

Joggz reviewed (commented):
Look Good to me. it's a good one but you are missing the last part
- The terms and condition and the copyright part
![lario1](https://user-images.githubusercontent.com/46860081/68034321-c1186580-fcc1-11e9-85ae-7c6191ad7df8.png)

Joggz committed:
made changes based on review:
- adds prop type validation to component

Joggz committed:
adds new line to end of each files

Joggz reviewed (approved):
It is Good to Go.

Mlg29 reviewed (approved):
LGTM

SEUNAGBEYE commented on dae-alright/src/components/FormInput/FormInput.component.jsx:
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
delete this file

SEUNAGBEYE replied:
and `formInput.style.csss`

SEUNAGBEYE reviewed (changes requested):
A few changes requested and this is good to go

SEUNAGBEYE commented:
Don't forget to rebase too.

LuthfulahiO commented on dae-alright/src/components/Footer/Footer.js:
```diff
@@ -0,0 +1,66 @@
+    return (
+        <footer className="deyAlrightFooterContainer">
+            <div className="logoDiv">
+             <img src="https://res.cloudinary.com/dq9utkjw2/image/upload/v1572357304/logo_zrlxud.png" alt="company-logo" className="footerLogo" />
```
Which company's logo? Alt should be descriptive, that is what the screen reader wants to read to a visually impaired person. Kindly update your alt property appropriately.

Joggz replied:
changed the alt attribute from company-logo to Dae-Alright Logo

LuthfulahiO commented on dae-alright/src/components/Footer/Footer.js:
```diff
@@ -0,0 +1,66 @@
+                         <li><a href="footer.js">Contact us</a></li>
+                        </ul>
+                    </div>
+                    <div className="socialImages">
```
![image](https://user-images.githubusercontent.com/20159012/68205484-bcf28d80-ffca-11e9-82e3-2562430b9ea3.png)
These guys are stretched, kindly fix.

LuthfulahiO commented on dae-alright/src/components/Footer/Footer.js:
```diff
@@ -0,0 +1,66 @@
+                    </div>
+                    <div className="socialImages">
+                        <img src="https://res.cloudinary.com/dtbjhs8a6/image/upload/v1571821215/my%20project/et9bmbe3mjhylwbfmree.png" alt="" />
+                        <img src="https://res.cloudinary.com/dtbjhs8a6/image/upload/v1571821216/my%20project/gix7baujoc1bqeczruru.png" alt="" />
```
Please add an alt attribute to all images

Joggz committed:
made changes on the review made:

Joggz committed:
made changes on the review made:

Joggz committed:
made changes on the review made:

Joggz committed:
made changes on the review made:

Joggz committed:
removed the Footer component from app.js

Joggz committed:
removed the Footer component from app.js

Joggz committed:
removed the Footer component from app.js

Joggz committed:
removed the Footer component from app.js

SEUNAGBEYE commented on dae-alright/src/components/Footer/Footer.js:
```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```
rename this file to `index.js`

SEUNAGBEYE commented on dae-alright/src/App.js:
```diff
@@ -1,14 +1,16 @@
     <div className="App">
-      <h1>dae-Alright</h1>
+      
+    
```
Use the footer here

SEUNAGBEYE commented on dae-alright/src/App.js:
```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Footer from './components/Footer/Footer'
```
import as `import Footer from './components/Footer`

Joggz committed:
mades changes from review given:
- changes Footer/Footer.js to Footer/index.js
- adds Footer component in the app.js for testing purpose

Joggz committed:
mades changes from review given:
- changes Footer/Footer.js to Footer/index.js
- adds Footer component in the app.js for testing purpose

John-pels committed:
I clear unwanted codes

Joggz committed:
Merge branch 'develop' of https://github.com/devcareer/Dae-Alright-Frontend into develop

Joggz committed:
made changes from review comment

Joggz committed:
made changes from review comment

Joggz committed:
resolves conflict

Joggz committed:
replaces Footer.js with index.js

Joggz committed:
adds term and condition paragraph

Joggz committed:
adds term and condition paragraph

Joggz committed:
adds  newline in Footer.scss

SEUNAGBEYE reviewed (approved):
LGTM

Joggz committed:
Merge branch 'develop' of https://github.com/devcareer/Dae-Alright-Frontend into develop

Joggz committed:
resolves conflict
